diff -ruN a/security/p5-Authen-SASL/Makefile b/security/p5-Authen-SASL/Makefile
--- a/security/p5-Authen-SASL/Makefile  2026-06-23 14:04:12.810991000 -0300
+++ b/security/p5-Authen-SASL/Makefile  2026-07-07 12:42:01.895481000 -0300
@@ -1,5 +1,5 @@
 PORTNAME=      Authen-SASL
-PORTVERSION=   2.1900
+PORTVERSION=   2.2000
 CATEGORIES=    security perl5
 MASTER_SITES=  CPAN
 PKGNAMEPREFIX= p5-
diff -ruN a/security/p5-Authen-SASL/distinfo b/security/p5-Authen-SASL/distinfo
--- a/security/p5-Authen-SASL/distinfo  2026-06-23 14:04:12.811051000 -0300
+++ b/security/p5-Authen-SASL/distinfo  2026-07-07 12:42:02.282716000 -0300
@@ -1,3 +1,3 @@
-TIMESTAMP = 1755072887
-SHA256 (Authen-SASL-2.1900.tar.gz) = be3533a6891b2e677150b479c1a0d4bf11c8bbeebed3e7b8eba34053e93923b0
-SIZE (Authen-SASL-2.1900.tar.gz) = 40345
+TIMESTAMP = 1783438922
+SHA256 (Authen-SASL-2.2000.tar.gz) = 8cdf5a7f185448b614471675dae5b26f8c6e330b62264c3ff5d91172d6889b99
+SIZE (Authen-SASL-2.2000.tar.gz) = 41529

